src/dist/replication/lib/replica_context.h

```diff
@@ -102,6 +102,20 @@ class primary_context
     dsn::task_ptr checkpoint_task;
 
     uint64_t last_prepare_ts_ms;
+
+    // Used for partition split
+    // child addresses who has been caught up its parent
+    std::unordered_set<dsn::rpc_address> caught_up_child;
+
+    // Used for partition split
+    // whether parent's write request should send to child synchronously
+    // if {sync_send_write_request} = true
+    // - parent should recevie prepare ack from child synchronously during 2pc
+    // if {sync_send_write_request} = false && replica {child_gpid}.get_app_id() > 0
```

Comment:
What does `replica {child_gpid}.get_app_id()` represent for?

Reply:
Replica class has a private variety `child_gpid`, whose structure is gpid. 
`replica {child_gpid}.get_app_id()` means `app_id` value of `child_gpid`. I will update this comment.